examples/grid-layout.adoc

```diff
@@ -0,0 +1,138 @@
+= Grid Layout
+:docinfo: private
+:source-highlighter: highlight.js
+:highlightjs-languages: clojure,scala
+:revealjs_hash: true
+
+[.columns]
+== 2 columns
+
+[.column]
+--
+* **Edgar Allen Poe**
+* Sheri S. Tepper
+* Bill Bryson
+--
+
+[.column]
+--
+Edgar Allan Poe (/poʊ/; born Edgar Poe; January 19, 1809 – October 7, 1849) was an American writer, editor, and literary critic.
+--
+
+[.columns.wrap]
+== Multi-columns with wrap
+
+[.column]
+--
+.Kotlin
+[source,kotlin]
+----
+fun main() {
+    println("Hello, World!")
+}
+----
+--
+
+[.column]
+--
+.Clojure
+[source,clojure]
+----
+(defn -main [& args]
+  (println "Hello, World!"))
+----
+--
+
+[.column]
+--
+.Scala
+[source,scala]
+----
+object HelloWorld {
+  def main(args: Array[String]): Unit = {
+    println("Hello, world!")
+  }
+}
+----
+--
+
+[.columns]
+== Columns with size
+
+[.column.is-one-third]
+--
+* **Kotlin**
+* Java
+* Scala
+--
+
+[.column]
+--
+Programming language for Android, mobile cross-platform
+and web development, server-side, native,
+and data science. Open source forever Github.
+--
+
+[.columns]
+== 3 columns
+
+[.column]
+* Java
+* **Kotlin**
+
+[.column]
+* Node
+* **Deno**
+
+[.column]
+* Ruby
+* **Crystal**
+
+
+[.columns.is-half.green.left]
+== !
+
+[.column.is-half]
+* Java
+* **Kotlin**
+
+[.column.is-half]
+* Ruby
+* **Crystal**
+
+== Languages
+
+We will go over languages
+
+[.columns]
+=== Kotlin
+
+[.column]
+[source,kotlin]
+----
+fun main() {
+    println("Hello, World!")
+}
+----
+
+[.column]
+--
+* Nice and compact
+* Works well on Android
+--
+
+[.columns]
+=== Clojure
+
+[.column]
+[source,clojure]
+----
+(defn -main [& args]
+  (println "Hello, World!"))
+----
+
+[.column]
+--
+* Lisp-like
+* Not sure why people like it
```

Comment:
Are you trolling Clojure? :laughing:

Reply:
Yeah, you should probably change that. I didn't know what to say about Clojure and I wanted two bullets. I haven't done Java or any JVM-language for 10+ years.